tests/baselines/reference/omitTypeTestErrors01.types

```diff
@@ -13,7 +13,7 @@ interface Foo {
 }
 
 export type Bar = Omit<Foo, "c">;
->Bar : { a: string; b: number; }
+>Bar : { b: number; a: string; }

```

Comment:
Why'd this change?

Reply:
It has to do with the order in which the literal types were materialized. Because we defer the `Omit` instantiations, we end up materializing the `"b"` and `"c"` literal types before we materialize `"a"`. Since `Omit` is non-homomorphic and since unions are ordered by type ID, this ends up flipping the properties.